### DISCUSSION TITLE
Why need two `SubmitWindowedPoSt` messages in one deadline?

### DISCUSSION THREAD

**Discussion by sokach-dev** [filecoin-project/lotus#11539] [General Q&A]
I found in the log that, once again, miner sent two messages in the windowpost message submission. Why is that?

![image](https://github.com/filecoin-project/lotus/assets/93634186/9cacfc0d-1d8c-4968-bd5d-27f6ca9a039c)

https://filfox.info/en/message/bafy2bzaceaaugr7dszmcute2u4uhwokmjpmvtvs4rsh3nzc7eff74uwrvodfk?t=1

https://filfox.info/en/message/bafy2bzacedfxofeeypl3cq2xfegnzgvvwnn7i4qlryuosw4m7ejeudp4ednsg?t=1

**Comment by TippyFlitsUK** [comment]
This explains why - https://github.com/filecoin-project/builtin-actors/pull/1436
